tracing/tracer.go

```diff
@@ -111,9 +111,8 @@ func (t *Tracer) Close() error {
 
 // NewTrace creates a new trace (top level local span).
 func (t Tracer) NewTrace(name string) *Span {
-	span := newSpan(&t, SpanTypeLocal)
-	span.name = name
-	span.traceID = rand.Uint64()
+	span := newSpan(&t, name, SpanTypeLocal)
+	span.NameeID = rand.Uint64()
```

Comment:
What is `NameeID`? It's not defined.

Reply:
I think this was just a wonky delete, it was supposed to delete one line but it looks like instead I deleted 1/2 of each line.